docs/pages/web/popover.js

```diff
@@ -525,6 +529,7 @@ function PopoverExample() {
 We recommend passing the following ARIA attribute to Popover for a better screen reader experience:
 
 - \`accessibilityLabel\`: describes the main purpose of a Popover for the screen reader. Should be unique and concise. For example, "Save to board" instead of "Popover".  It populates [aria-label](https://w3c.github.io/aria-practices/#dialog_roles_states_props).
+- \`accessibilityDismissButtonLabel\`: describes the purpose of the dismiss button on Popover for the screen reader. Should be clear and concise. For example, "Close board selection popover" instead of "Close".
```

Comment:
Is "popover" a broad enough concept for user to understand what it is?

Reply:
yeah, I think screen reader users will understand it